ios/FluentUI/List/FluentListSection.swift

```diff
@@ -76,3 +76,12 @@ public extension FluentListSection where SectionHeaderContent == EmptyView {
         self.footer = footer
     }
 }
+
+public extension FluentListSection where SectionHeaderContent == Text, SectionFooterContent == EmptyView {
```

Comment:
would it also be beneficial to do something similar for the footer? But would just have to change the init to not use `_` to disambiguate the two.